pkg/reconciler/v1alpha1/builder/builder_test.go

```diff
@@ -12,12 +12,11 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgotesting "k8s.io/client-go/testing"
 	"k8s.io/client-go/tools/record"
-	"knative.dev/pkg/apis"
-	duckv1alpha1 "knative.dev/pkg/apis/duck/v1alpha1"
 	"knative.dev/pkg/controller"
 	rtesting "knative.dev/pkg/reconciler/testing"
 
 	"github.com/pivotal/kpack/pkg/apis/build/v1alpha1"
+	kpackcore "github.com/pivotal/kpack/pkg/apis/core/v1alpha1"
```

Comment:
maybe we use `buildv1alpha1` and `corev1alpha1` and `expv1alpha1` for our various types? Just a suggestion

Reply:
That makes sense to me.

Reply:
I like that. Would it be confusing with k8s/core?

Reply:
i know what you mean, but `kpackcorev1alpha1` is also the name of my dog, and i would be constantly confused 😛 
i am slightly comfortable with this because i dont think `k8s/core` has a v1alpha1

Reply:
I didn't know you had a dog.

Reply:
to be completely honest, i learnt that about myself 5 minutes ago as well